crates/oxc_semantic/src/builder.rs

```diff
@@ -69,6 +72,8 @@ impl<'a> SemanticBuilder<'a> {
         program: &'a Program<'a>,
         trivias: &Rc<Trivias>,
     ) -> SemanticBuilderReturn<'a> {
+        self.trivias = Rc::clone(trivias);
```

Comment:
Set trivias when we call `.build()`, that way we don't need to update the `Visit` trait